PeerioTechnologies/peerio-desktop
# fix: Files view UI updates, bug fixes, styling fixes

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/7950/desktop-tooltip-for-pending-files
https://app.clubhouse.io/peerio/story/8177/desktop-file-list-ui-fixes
https://app.clubhouse.io/peerio/story/9071/desktop-files-view-download-progress-bar-spacing-is-wrong-due-to-dom-changes-in-newfs
https://app.clubhouse.io/peerio/story/9077/desktop-in-files-view-bulk-actions-do-not-show-when-selecting-items-from-search-results

https://github.com/PeerioTechnologies/peerio-icebear/pull/228

#### Testing instructions  
Everything here is in the main Files view.

* for a non-migrated file, hovering on the "Pending" badge shows a tooltip with info in it
* select-all checkbox at the top should only appear once when at least one file has already been bulk selected (or the user hovers over the files header)
* clicking on the three dots menu of any file or folder should keep the row's hover state (blue background) until the menu is closed/canceled
* three dots menu is no longer disabled when a file/folder is selected by checkbox
* icons in file rows now show proper hover colours even when you're hovering on the far left/right of the row
* download/upload/share progress bars extend the full width of the row again
* when bulk selecting files/folders from search results, bulk actions buttons show up now

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
